bin/main.rb

```diff
@@ -0,0 +1,5 @@
+# A file which has a shebang line as its first line is
+# granted execute permission.
+
+# !/usr/bin/env ruby
```

Comment:
- [ ] Make sure that this line is the first line of your document. Please notice that this is not a regular comment, this is a "hidden" command, so it doesn't need a space after the `#`. Rubocop won't complain about the space when it is recognized as a command (in the first line of the document)